app/connectors/httpParsers/ScratchHttpParser.scala

```diff
@@ -17,18 +17,18 @@
 package connectors.httpParsers
 
 import models.errors.{InternalServerError, InvalidProcessError}
-import models.{RequestOutcome, SaveScratchSubmissionResponse}
+import models.{RequestOutcome, ScratchResponse}
 import play.api.Logger
 import play.api.http.Status._
 import uk.gov.hmrc.http.HttpReads
 
-object SaveScratchProcessHttpParser extends HttpParser {
+object ScratchHttpParser extends HttpParser {
 
-  val logger: Logger = Logger(SaveScratchProcessHttpParser.getClass)
+  val logger: Logger = Logger(ScratchHttpParser.getClass)
```

Comment:
As above I think we Mac would prefer we don't use class specific loggers.

Reply:
Hi Chris, have you pushed an update for this?

Reply:
updated in recent commit